src/custom/hooks/useFetchFile.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react'
+
+export default function useFetchFile(filePath: string) {
```

Comment:
nit: in theory this is useFechContent :P
useFechFile could download the file, this one you want the content of any http resource, it might even not be a file, just a GET endpoint. 

anyways, just nitpicking

Reply:
Yeah that's how I had it before...`useFetchContent`... and exporting {content, error] but `content` was interfering a bit with other variable names where I was importing the hook (I know I can use `import as ..`). So in the end it became useFetchFile :) - But happy to change again.